tests/test_autocomplete.py

```diff
@@ -1,7 +1,7 @@
 # test_autocomplete.py
 
 from customUrwidClasses import CommandBar
-from commandChanVim import urwidView
+from ..commandChanVim import urwidView
```

Comment:
I think this should be just commandChanVim bc tests are run from the top level directory. See output of travis